scalabel/vis/label.py

```diff
@@ -475,6 +480,35 @@ def draw_graph(self, labels: List[Label]) -> None:
                     )
                     self.ax.add_patch(result[0])
 
+    def draw_rle(
+        self,
+        image: NDArrayU8,
+        labels: List[Label],
+        alpha: float = 0.5,
+    ) -> None:
+        """Draw RLE."""
+        combined_mask = np.zeros(image.shape)
+
+        for label in labels:
+            if not label.rle:
+                continue
+
+            color = self._get_label_color(label) * 255  # type: NDArrayF64
+            bitmask = rle_to_mask(label.rle)
+            mask = np.repeat(bitmask[:, :, np.newaxis], 3, axis=2)
+
+            # Non-zero values correspond to colors for each label
+            combined_mask = np.where(
+                mask, color.astype(np.uint8), combined_mask
+            )
+
+        if combined_mask is None:
+            return
+        self.ax.imshow(
+            np.where(combined_mask > 0, combined_mask.astype(np.uint8), image),
```

Comment:
This may not work when the height and the width of the UI is changed, since image will be of original resolution. Can you test with modifying the height and width and seeing if it still works? See [UIConfig](https://github.com/scalabel/scalabel/blob/master/scalabel/vis/label.py#L57) for the config and [`draw_image`](https://github.com/scalabel/scalabel/blob/master/scalabel/vis/label.py#L222) for the resizing of the image.